pkgs/development/libraries/nghttp3/default.nix

```diff
@@ -14,7 +14,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-pV1xdQa5RBz17jDINC2uN1Q+jpa2edDwqTqf8D5VU3E=";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkg-config cunit file ];
+  nativeBuildInputs = [ autoreconfHook pkg-config file ];
+  checkInputs = [ cunit ncurses ];
```

Comment:
`cunit` requires `ncurses` to be specified as co-dependencies on Darwin.

Reply:
Should it go to `cunit.propagatedBuildInputs` then? Also, is this because of python?

Reply:
My bad, I got confused about the origin of this requirement. I mistakenly recalled this as a Nixpkgs thing, but it was actually a requirement of the nghttp3/ngtcp2 build script:

https://github.com/ngtcp2/nghttp3/blob/484c40cc3b7414e484628e5e12532199fd2e3cb9/configure.ac#L122-L132

Reply:
I did a quick search on the Nixpkgs GitHub issues and the requirement doesn't seem commonplace.